docs/developers/developing_iree/releasing.md

```diff
@@ -0,0 +1,83 @@
+# Releasing
+
+IREE cuts automated releases via a workflow that is
+[triggered daily](https://github.com/openxla/iree/blob/main/.github/workflows/schedule_candidate_release.yml).
+The only constraint placed on the commit that is released is that it has passed
+all CI checks. These are published on GitHub with the "pre-release" status. For
+debugging this process, see the
+[Debugging releases playbook](../debugging/releases.md).
+
+We periodically promote one of these candidates to a "stable" release by
+removing the "pre-release" status. This makes it show up as a "latest" release
+on GitHub. We also push the Python packages for this release to PyPI.
+
+The criteria for selecting this candidate is a bit more involved.
+
+
+## Coupling to the Google Integrate
+
+The Google team that manages these stable releases at the moment is also
+responsible for integrating the IREE source code into Google's monorepo. For
+convenience, we select a candidate pre-release, attempt to integrate it into
+Google's monorepo and then promote it to stable if that was successful without
+cherry picks that would affect the quality of the release (because they wouldn't
+be present in the promoted version).
+
+This gives some additional validation to the release because it is stress-tested
+running in a different environment and we not-infreuqently catch some issues. We
```

Comment:
typo, fix or just remove the double negative

```suggestion
This gives some additional validation to the release because it is stress-tested
running in a different environment and we periodically catch some issues. We
```

Reply:
Hmm I think periodically means something different :grin: 

```suggestion
This gives some additional validation to the release because it is stress-tested
running in a different environment and we not-infrequently catch some issues. We
```

Reply:
"not infrequently" == "frequently" == "periodically"? IDK... "occasionally", "sometimes", "rarely"?

Reply:
It's not *infrequent* but that doesn't mean it's *frequent*. I think sometimes is closest, but I think the emphasis is different. "periodically" focuses on it happening at regular intervals which isn't the case here at all. Anyway, I'm going to leave this unless you care a lot :-)